Source/Archiver.h

```diff
@@ -63,7 +63,7 @@ SOFTWARE
 
 #define	ARCHIVER_SETTINGS_FILE_PATH		"/boot/home/config/settings/"
 #define	ARCHIVER_SETTINGS_FILE			"archiver.settings"
-#define	ARCHIVER_RULES_FILE_PATH		"/boot/home/config/etc/"
+#define	ARCHIVER_RULES_FILE_PATH		"/boot/home/config/non-packaged/etc/"
```

Comment:
user should paste the rules file to that location but the program don't use the file on that location as Pete Goodeve said ,  it using rules file in " B_SYSTEM_ETC_DIRECTORY" . and also user should paste rules file on their but if I comment that  path it won't compile :/ but when i delete the rules file from non-packaged nothing changed :/